core/modules/system/system.admin.inc

```diff
@@ -234,14 +234,27 @@ function system_themes_admin_form($form, &$form_state, $theme_options) {
   // Administration theme settings.
   $form['admin_theme'] = array(
     '#type' => 'fieldset',
-    '#title' => t('Administration theme'),
+    '#title' => t('Administration theme settings'),
+  );
+  $form['admin_theme']['admin_theme_same_as_default'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Always use the default theme as the administration theme'),
```

Comment:
Do we need the "Always" here? What about just `Use the default theme as the administration theme`.

Reply:
...done 👍